devops/deploy/spin-down-script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Now fully implemented spinning down through the port_list.txt
+IP=127.0.0.1
+declare -a PORTS=()
+while IFS= read -r line
+do
+	PORTS+=("$line")
+done < "port_list.txt"
+
+# Iterate through the list, killing each controlling process while looping
+for port in ${PORTS[*]}
+do
+	echo "Killing processes at $IP:$port"
+	temp=$IP:$port
+	kill -15 $(netstat -anp | grep $temp | awk '{print $6}' | awk -F/ '{print $1}')
```

Comment:
This will work, so not requesting a change at this time. Note that `lsof` with `-t` will make this easier though.

```
2024-01-18 16:01:00 ~
1002 | 0 $ lsof -t -i@127.0.0.1:42422
24877
2024-01-18 16:01:34 ~
1003 | 0 $
```